test/scripts/jenkins_firefox_smoke.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -e
+trap 'node "$KIBANA_DIR/src/dev/failed_tests/cli"' EXIT
+
+node scripts/build --debug --oss;
+linuxBuild="$(find "$KIBANA_DIR/target" -name 'kibana-oss-*-linux-x86_64.tar.gz')"
+installDir="$PARENT_DIR/install/kibana"
+mkdir -p "$installDir"
+tar -xzf "$linuxBuild" -C "$installDir" --strip=1
+
+export TEST_BROWSER_HEADLESS=1
+
+checks-reporter-with-killswitch "Firefox smoke test" \
+  node scripts/functional_tests \
+    --bail --debug \
```

Comment:
Is this why I'm getting debug output (without the `--debug` flag) when I run;

`node scripts/functional_test_runner  --config test/functional/config.firefox.js --include-tag smoke`

Reply:
As we discussed, debug is `true` by default since 9122511370431fc8486f146046aff1a327cfd032